dartagnan/src/main/java/com/dat3m/dartagnan/program/Program.java

```diff
@@ -123,12 +123,12 @@ public Collection<INonDet> getConstants() {
     /**
      * Iterates all events in this program.
      *
-     * @return {@code cId}-ordered complete sequence of all events in this program.
+     * @return {@code globalId}-ordered complete sequence of all events in this program.
      */
-    public List<Event> getEvents() {
+    public List<Event> getThreadEvents() {
```

Comment:
The name suggest this is expected to be called after threads were built. What if the developer calls this before? If we intended to be used only after building threads, we just add some check to detect wrong usage

Reply:
I can add a check. If someone calls it before, an empty list gets returned right now.